activity_browser/ui/wizards/db_import_wizard.py

```diff
@@ -1062,7 +1063,7 @@ def run(self):
 class EcoinventVersionPage(QtWidgets.QWizardPage):
     def __init__(self, parent=None):
         super().__init__(parent)
-        self.wizard = self.parent()
+        self.wizard: "DatabaseImportWizard" = self.parent()
```

Comment:
In this context, looking at it, the most I can assume is `wizard: any | None`.
Just because there is one usage, doesn't mean that it can be inferred that is indeed `DatabaseImportWizard`.
My 2 cents, if it is bothersome i can remove it.